### PR TITLE
Update table label filter to reuse .data-toolbar .data-toolbar-filter and extend on top of table

### PR DESCRIPTION
**Pull request by sg00dwin**
Fixes https://github.com/openshift/origin-web-console/issues/1824

tested on Chrome, Safari, Firefox, IE

screen shot 2017-07-12 at 11 06 59 am
screen shot 2017-07-12 at 11 07 26 am

screen shot 2017-07-12 at 11 24 56 am

28127725-85c9aed4-66fb-11e7-85de-b07a004252bc

### CODE REVIEW

**Comment by spadgett** [comment]
This will conflict with #1826 :/

**Comment by spadgett** [comment]
I can rebase #1826 on this one. I think I can reuse the style.

File: dist/styles/vendor.css [hunk 1]
```diff
@@ -178,7 +178,7 @@ kubernetes-container-terminal{position:relative;display:block}
 kubernetes-container-terminal .terminal{font-family:"Monospace Regular","DejaVu Sans Mono",Menlo,Monaco,Consolas,monospace;font-size:10px;color:#F0F0F0;text-align:left;outline:0;background-color:#000;border:1px solid #000;padding:10px;display:inline-block;line-height:1em}
 @media (min-width:568px){kubernetes-container-terminal .terminal{font-size:12px}
 }
-kubernetes-container-terminal .terminal-cursor{color:#000;background:#f0f0f0}
+kubernetes-container-terminal .terminal-cursor{background:#f0f0f0;color:#000;position:absolute}
```
Comment (rhamilto) [inline dist/styles/vendor.css]: This is invalid.  Make sure you update your dependencies and rebuild.

**Comment by sg00dwin** [comment]
@spadgett updated pr to remove inadvertent vendor changes

**Comment by spadgett** [comment]
[merge][severity: bug]

**Pull request merged**